t/24-worker-engine.t

```diff
@@ -18,10 +18,14 @@ use warnings;
 
 use FindBin;
 use lib ("$FindBin::Bin/lib", "$FindBin::Bin/../lib");
+use Test::Exception;
 use Test::Fatal;
 use Test::More;
+use Test::Output qw(stdout_like stderr_like);
 use Test::Warnings;
 use OpenQA::Worker;
+use Test::MockModule;
+use Test::MockObject;
```

Comment:
Do we need these modules?

Reply:
Right. These are only needed in https://github.com/os-autoinst/openQA/pull/2503 in another commit. I will change this, thx.